src/Proto.Actor/Supervision.cs

```diff
@@ -23,8 +23,8 @@ public enum SupervisorDirective
     public interface ISupervisor
     {
         IReadOnlyCollection<PID> Children { get; }
-        void EscalateFailure(PID who, Exception reason);
-        void RestartChildren(params PID[] pids);
+        void EscalateFailure(Exception reason, PID who);

```

Comment:
I switched this around to keep it consistent with RestartChildren, as that method uses params, which need to come last